lang/cs/Org.Apache.REEF.IMRU.Examples/README.md

```diff
@@ -0,0 +1,35 @@
+# Running IMRU Examples
+
+The easiest way to run IMRU examples is from command line.
+Build C# code using your preferred method.
+
+## Running on local runtime
+
+Navigate to `<reef>\bin\x64\Debug\Org.Apache.REEF.IMRU.Examples`. 
+
+Run `.\Org.Apache.REEF.IMRU.Examples.exe` with first parameter set to `false`.
+
+## Running on Yarn
+
+Log in to a machine in Yarn cluster from which you'll run the job.
+
+Copy the contents of `<reef>\bin\x64\Debug\Org.Apache.REEF.IMRU.Examples` to this machine.
+
+Run `.\Org.Apache.REEF.IMRU.Examples.exe` with first parameter set to `true`.
+
+## Command line parameters
+
+Command line parameters are parsed based on their position in the list of parameters, so if you need to pass a non-default value for a parameter, you have to pass explicit values for all preceding parameters.
+
+1. run on yarn (default `false`): if `true`, run the test on Yarn, otherwise run the test on local runtime.
+2. number of nodes (default `2`): number of nodes on which IMRU job will run. This number includes the master node, i.e. the number of mappers will be one less than the value you pass.
+3. start port (default `8900`): the start of port range.
```

Comment:
Maybe add something like:

`
Many clusters have firewall configurations which only allow processes to open ports in specific ranges. These are specified here as a start and end of the port range. REEF will pick a random, available port from that range for its communications.`